modules/eks-cluster/outputs.tf

```diff
@@ -71,6 +71,16 @@ output "oidc_provider_arn" {
   description = "Amazon Resource Name of the OIDC provider for the EKS cluster. Allows to add additional IRSA mappings"
 }
 
+output "aws_caller_identity_account_id" {
+  value       = data.aws_caller_identity.current.account_id
+  description = "Account ID of the current AWS account"
+}
```

Comment:
Maybe it's used or mentioned somewhere else but not sure the output makes sense.
As you used below already the default is just `data.aws_caller_identity.current.account_id` and not sure the EKS module is the right source for this information.

Reply:
all good then 👍, didn't immediately see it being used or overlooked it.